recipes/kaitai_struct_cpp_stl_runtime/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class KaitaiStructCppStlRuntimeConan(ConanFile):
+    name = "kaitai_struct_cpp_stl_runtime"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://kaitai.io/"
+    description = "kaitai struct c++ runtime library"
+    topics = ("parsers", "streams", "dsl")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    options = {
+        "with_zlib": [True, False],
+        "with_iconv": [True, False]
+    }
+    default_options = {
+        "with_zlib": False,
+        "with_iconv": False
+    }
+
```

Comment:
```suggestion
    short-paths = True
```

Because Windows build failed due maximum path length.

https://docs.conan.io/en/latest/reference/conanfile/attributes.html#short-paths